workflow/common/common.go

```diff
@@ -132,6 +131,8 @@ const (
 	EnvVarProgressFileTickDuration = "ARGO_PROGRESS_FILE_TICK_DURATION"
 	// EnvVarProgressFile is the file watched for reporting progress
 	EnvVarProgressFile = "ARGO_PROGRESS_FILE"
+	// EnvVarDefaultRequeueTime is the default requeue time for Workflow Informers. For more info, see rate_limiters.go
+	EnvVarDefaultRequeueTime = "DEFAULT_REQUEUE_TIME"
```

Comment:
nice, only request `ARGO_DEFAULT_REQUEUE_TIME`

Reply:
@alexec I have not created a new variable here, but simply reused the existing one. My fear is that changing the variable name will break users who have `DEFAULT_REQUEUE_TIME` hard wired. I can always accept both if necessary?

Reply:
ah - prior art - ok

Reply:
thought - do we want to make this independently configurable? that way, any problems, we can change them independently

Reply:
Sorry, not sure what you mean by independently configurable?

Reply:
I think we should be able to cater for the agent going crazy and we want to reduce its patch periodicity, but not the controller's re-queue time. That would mean two configuration items.

Reply:
Sure thing, I can allow for two configuration items. If no configuration is found specifically for agent patch periodicity, I will fall back to controller's re-queue time as default.